Sources/Subprocess/IO/AsyncIO+Windows.swift

```diff
@@ -174,10 +183,10 @@ final class AsyncIO: @unchecked Sendable {
         }
         // Post status to shutdown HANDLE
         PostQueuedCompletionStatus(
-            ioPort,         // CompletionPort
-            0,              // Number of bytes transferred.
-            shutdownPort,   // Completion key to post status
-            nil             // Overlapped
+            ioPort, // CompletionPort
+            0, // Number of bytes transferred.
+            shutdownPort, // Completion key to post status
+            nil // Overlapped
```

Comment:
Personally I find the previous version more readable.

Reply:
Me too, but there's no setting for swift-format to recognize that... There's also no "disable linting for the next n lines" either so we are out of options for now. I think `swift-format` desperately  need a `ignore next N lines` in addition to `ignore this file`.